### ISSUE TITLE
Repository `.meta` file Unexpectedly Erased

### ISSUE THREAD

**Issue by coddingtonbear** [coddingtonbear/inthe.am#307]
On at least two occasions over the last few weeks, I've received reports from users receiving a 500 error when attempting to log-in to the site.  During investigation, I discovered that there were uncommitted changes in their task store including changes that deleted their `.meta` file.   I have not yet tried to track down a corresponding error in Sentry, but will investigate later.

**Comment by jpretori** [comment]
I'm getting a 500 error when first trying to log into the site:

> An error was encountered while attempting to gather user information:
Server Error (500)

  This is with a brand new installation of taskwarrior on a new-ish Ubuntu 18.04 LTS system.  The only thing I've done in Taskwarrior is to have it autocreate .taskrc.
I'm using Firefox on the Ubuntu machine.

About an hour ago, I got the same error in much the same situation (very new taskwarrior setup), but using Chrome on MacOSX.  On MacOS, when I switched to Brave Browser, the problem disappeared, and I was able to log in and complete setup of the link between my local taskwarrior setup and inthe.am.
